### PR TITLE
Add cache parameter and change default values for LIBSVM models

### PR DESCRIPTION
**Pull request by OkonSamuel**
This closes issue #253.
This also addresses issue #241

### CODE REVIEW

File: src/builtins/Transformers.jl [hunk 1]
```diff
@@ -301,12 +301,12 @@ MLJBase.inverse_transform(transformer::UnivariateStandardizer, fitresult, w) =
...
-standardized are the `Continuous` features named in `features`
-(`ignore=false`) or `Continuous` features not named in `features`
-(`ignore=true`). To allow standarization of `Count` or `OrderedFactor`
-features as well, set the appropriate flag to true.
+tabular data.  If `features` is unspecified then all columns `v`
+having Continuous element scitype are standardized. Otherwise, the
+features standardized are the `Continuous` features named in
+`features` (`ignore=false`) or `Continuous` features not named in
+`features` (`ignore=true`). To allow standarization of `Count` or
+`OrderedFactor` features as well, set the appropriate flag to true.
```
Comment (ablaom) [inline src/builtins/Transformers.jl:310]: This looks like a reversal of earlier but valid (minor) correction.

File: src/LIBSVM.jl [hunk 2]
```diff
@@ -298,7 +297,7 @@ function EpsilonSVR(
         ,gamma
         ,epsilon
         ,cost
-        ,cache_size
+        ,cachesize
```
Comment (ablaom) [inline src/LIBSVM.jl:301]: Generally, we try to stick with variable names that coincide with the source package documentation. I guess you checked this?

File: src/LIBSVM.jl [hunk 3]
```diff
@@ -12,6 +12,7 @@ import MLJModelInterface: Table, Continuous, Count, Finite, OrderedFactor,
 const MMI = MLJModelInterface
 
 import ..LIBSVM
+using ..Statistics
```
Comment (ablaom) [inline src/LIBSVM.jl:16]: Why not simply `using Statistics`?

**Comment by ablaom** [comment]
Looks good to me, many thanks. But now our Mac MKL errors have spread to all julia versions 😢 

Possibly a result of a recent scikit-learn update.  

@tlienart Any thoughts?

**Comment by tlienart** [comment]
let's just drop osx from CI for now

File: src/builtins/Transformers.jl [hunk 4]
```diff
@@ -193,7 +193,7 @@ function MLJBase.fit(transformer::UnivariateDiscretizer, verbosity::Int, X)
     even_quantiles = clipped_quantiles[1:2:(2*n_classes-1)]
 
     # determine optimal reference type for encoding as categorical:
-    R = reftype(categorical(1:n_classes, compress=true)
+    R = reftype(categorical(1:n_classes, compress=true))
     output_prototype = categorical(R(1):R(n_classes), compress=true, ordered=true)
```
Comment (ablaom) [inline src/builtins/Transformers.jl:197]: Thanks.

**Pull request merged**